backend/src/test/java/io/debezium/configserver/util/PostgresInfrastructureTestProfile.java

```diff
@@ -0,0 +1,18 @@
+package io.debezium.configserver.util;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class PostgresInfrastructureTestProfile implements QuarkusTestProfile {
+    @Override
+    public List<TestResourceEntry> testResources() {
+        ArrayList<TestResourceEntry> resourceEntries = new ArrayList<>(1) {};
```

Comment:
Any specific reason for not using the much shorter `Arrays.asList(...)`?

Reply:
just the way how it gathered... will use `Collections.singletonList(...)`